WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -23,6 +23,23 @@
     end
   end
 
+  context "delegates" do
+    it "delegates for future comps must be delegates" do
```

Comment:
Discussed with @larspetrus offline, and we decided to go with "delegates for future comps must be current delegates".